### PR TITLE
Custom datasource adapter

### PR DESCRIPTION
**Pull request by marcinkp**
Instead mvn archetype, providing simple data source adapter implementation for knotx-data-bridge.
In addition docker database instance for tutorial.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: acme-sample-database/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM java:openjdk-8-jre-alpine
```
Comment (tomaszmichalak) [inline acme-sample-database/Dockerfile]: Please move those files to the `acme-db-datasource-adapter`.
```
- knotx-example-project
  - knotx-databrige
    - acme-db-datasource-adapter
      - Dockerfile
      - books.script
      - etc.
```

File: acme-datasource-adapter/build.gradle [hunk 2]
```diff
@@ -0,0 +1,70 @@
+plugins {
```
[the remark this answers is not in the document]
Comment (marcinkp) [reply acme-datasource-adapter/build.gradle]: I created PR for  Knot.x Data bridge. Still haven't done the the build of starter kit

File: README.md [hunk 3]
```diff
@@ -92,6 +94,7 @@ Knot.x works in two modes:
     - [http://localhost:8092/content/simple.html](http://localhost:8092/content/simple.html)
     - [http://localhost:8092/content/login/step1.html](http://localhost:8092/content/login/step1.html)
     - [http://localhost:8092/content/multiple-forms.html](http://localhost:8092/content/multiple-forms.html)
+    - [http://localhost:8092/content/db-books.html](http://localhost:8092/content/db-books.html) _Note_: to have it working you need to run local DB. Details can be found [Readme](https://github.com/Knotx/knotx-example-project/tree/master/knotx-data-bridge/acme-db-datasource-adapter)
```
Comment (tomaszmichalak) [inline README.md]:
```suggestion
    - [http://localhost:8092/content/db-books.html](http://localhost:8092/content/db-books.html) _Note_: to have it working you need to run your local HSQLDB instance. More details can be found [here](https://github.com/Knotx/knotx-example-project/tree/master/knotx-data-bridge/acme-db-datasource-adapter). 
```

File: build.gradle [hunk 4]
```diff
@@ -1,7 +1,25 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-allprojects {
+plugins {
```
Comment (tomaszmichalak) [inline build.gradle]: Can we please move your changes to `gradle/knotx-common.gradle` file. Then you can reference this file in all required modules.

The example: https://github.com/Cognifide/gradle-aem-multi/blob/master/gradle/common.gradle.kts
https://github.com/Cognifide/gradle-aem-multi/blob/master/aem/build.gradle.kts#L35

**Comment by tomaszmichalak** [comment]
Invalid.

**Pull request closed without merging**